src/custom/components/analytics/GoogleAnalyticsReporter.tsx

```diff
@@ -0,0 +1,30 @@
+import { useActiveWeb3React } from '@src/hooks'
+import { useEffect } from 'react'
+import ReactGA from 'react-ga'
+import { RouteComponentProps } from 'react-router-dom'
+import { NETWORK_LABELS as NETWORK_LABELS_HEADER } from 'components/Header'
+import { ChainId } from '@uniswap/sdk'
+
+const NETWORK_DIMENSION = 'dimension1'
+export const NETWORK_LABELS: { [chainId in ChainId]?: string } = {
+  ...NETWORK_LABELS_HEADER,
+  [ChainId.MAINNET]: 'Mainnet' // The header don't has Mainet
```

Comment:
```suggestion
  [ChainId.MAINNET]: 'Mainnet' // The header doesn't have Mainet
```

Reply:
adding a suggestion to typo fix :D

```suggestion
  [ChainId.MAINNET]: 'Mainnet' // The header doesn't have Mainnet
```